packages/fulfillment/src/services/fulfillment-module-service.ts

```diff
@@ -235,6 +252,11 @@ export default class FulfillmentModuleService<
       return []
     }
 
+    const rules = data_.flatMap((d) => d.rules)
```

Comment:
Maybe it's nice to validate the individual requests as well, as you can have
[[], [rule]]
and I assume an empty ruleset isn't really valid? Maybe it is, just wanted to bring it up

Reply:
yes am empty rule set is accepted, at least from my understanding, the shipping options could be filtered only by its location (geo zone) but has not other rules to apply for it